pdns/rcpgenerator.cc

```diff
@@ -607,6 +718,61 @@ void RecordTextWriter::xfrHexBlob(const string& val, bool)
   }
 }
 
+void RecordTextWriter::xfrSvcParamKeyVals(const set<SvcParam>& val) {
+  for (auto const &param : val) {
+    if (!d_string.empty())
```

Comment:
is the intent "d_string is empty" or "is this not the first value"? if the latter, i'd suggest a `bool first` instead :)